src/batch/src/executor/join/local_lookup_join.rs

```diff
@@ -135,6 +136,7 @@ impl<C: BatchTaskContext> InnerSideExecutorBuilder<C> {
                 }),
             }),
             epoch: Some(self.epoch.clone()),
+            tracing_context: TracingContext::from_current_span().to_protobuf(),
```

Comment:
`LocalLookupJoin` might cause a lot of tracing spans since it depends on the outer side's data size, but we have auto query mode which could reduce these cases by choosing the `DistributedLookupJoin`, so LGTM.